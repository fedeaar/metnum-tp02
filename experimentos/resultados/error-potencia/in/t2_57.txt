7.009225000000000000e+06 1.245539000000000000e+06 -7.466840000000000000e+05 -1.323990000000000000e+06 1.365410000000000000e+06 5.330300000000000000e+05 -4.176720000000000000e+05 -1.532287000000000000e+06 2.147860000000000000e+06 -2.614469000000000000e+06 2.197126000000000000e+06 1.965468000000000000e+06 2.336184000000000000e+06 -2.882240000000000000e+06 -8.165580000000000000e+05 1.426460000000000000e+05 5.226100000000000000e+05 1.222390000000000000e+06 8.798570000000000000e+05 5.588720000000000000e+05
1.245539000000000000e+06 6.559379000000000000e+06 -1.822047000000000000e+06 8.740590000000000000e+05 6.447540000000000000e+05 1.687900000000000000e+05 1.925126000000000000e+06 -2.795907000000000000e+06 1.497553000000000000e+06 -2.954812000000000000e+06 9.719040000000000000e+05 6.823380000000000000e+05 1.355380000000000000e+06 -3.490950000000000000e+05 3.693500000000000000e+05 -2.688972000000000000e+06 -2.184907000000000000e+06 -2.521874000000000000e+06 -3.889820000000000000e+05 -1.569430000000000000e+06
-7.466840000000000000e+05 -1.822047000000000000e+06 8.234913000000000000e+06 -1.601219000000000000e+06 1.257000000000000000e+04 1.197871000000000000e+06 -7.171660000000000000e+05 2.946398000000000000e+06 -4.569650000000000000e+05 1.455844000000000000e+06 -9.718790000000000000e+05 -8.947440000000000000e+05 1.315040000000000000e+05 -1.659880000000000000e+06 2.445942000000000000e+06 5.959470000000000000e+05 1.692314000000000000e+06 1.053343000000000000e+06 1.667551000000000000e+06 2.019084000000000000e+06
-1.323990000000000000e+06 8.740590000000000000e+05 -1.601219000000000000e+06 7.803211000000000000e+06 -1.350700000000000000e+04 2.545430000000000000e+06 -3.217880000000000000e+05 -1.338205000000000000e+06 2.557760000000000000e+05 -1.295396000000000000e+06 4.829640000000000000e+05 2.111950000000000000e+05 2.041910000000000000e+06 1.123700000000000000e+06 1.132003000000000000e+06 -1.125240000000000000e+05 -3.881438000000000000e+06 -7.713710000000000000e+05 -3.606625000000000000e+06 -8.386970000000000000e+05
1.365410000000000000e+06 6.447540000000000000e+05 1.257000000000000000e+04 -1.350700000000000000e+04 7.093490000000000000e+06 -2.358101000000000000e+06 8.467790000000000000e+05 -1.370680000000000000e+06 3.058124000000000000e+06 -7.393030000000000000e+05 7.133100000000000000e+05 2.338100000000000000e+04 1.188654000000000000e+06 -2.286279000000000000e+06 -6.995700000000000000e+04 -2.171374000000000000e+06 -3.498390000000000000e+05 2.945100000000000000e+05 -2.027497000000000000e+06 -1.897100000000000000e+06
5.330300000000000000e+05 1.687900000000000000e+05 1.197871000000000000e+06 2.545430000000000000e+06 -2.358101000000000000e+06 8.132930000000000000e+06 4.433300000000000000e+04 -6.229740000000000000e+05 -1.831600000000000000e+05 -1.876736000000000000e+06 1.443500000000000000e+05 -8.126180000000000000e+05 1.841048000000000000e+06 -2.065816000000000000e+06 1.776824000000000000e+06 2.228190000000000000e+05 -1.572740000000000000e+05 -3.136020000000000000e+05 -2.257661000000000000e+06 -1.345754000000000000e+06
-4.176720000000000000e+05 1.925126000000000000e+06 -7.171660000000000000e+05 -3.217880000000000000e+05 8.467790000000000000e+05 4.433300000000000000e+04 7.784259000000000000e+06 -1.645097000000000000e+06 1.875924000000000000e+06 2.058844000000000000e+06 -3.032070000000000000e+05 2.319858000000000000e+06 -1.825408000000000000e+06 2.962940000000000000e+05 5.189110000000000000e+05 -2.467026000000000000e+06 -1.435902000000000000e+06 2.269541000000000000e+06 1.247689000000000000e+06 -1.782525000000000000e+06
-1.532287000000000000e+06 -2.795907000000000000e+06 2.946398000000000000e+06 -1.338205000000000000e+06 -1.370680000000000000e+06 -6.229740000000000000e+05 -1.645097000000000000e+06 6.410190000000000000e+06 -1.557137000000000000e+06 6.160880000000000000e+05 5.155980000000000000e+05 -1.392543000000000000e+06 -1.632003000000000000e+06 5.809100000000000000e+04 3.054840000000000000e+05 1.129953000000000000e+06 2.437259000000000000e+06 3.097930000000000000e+05 3.499538000000000000e+06 3.597444000000000000e+06
2.147860000000000000e+06 1.497553000000000000e+06 -4.569650000000000000e+05 2.557760000000000000e+05 3.058124000000000000e+06 -1.831600000000000000e+05 1.875924000000000000e+06 -1.557137000000000000e+06 5.597586000000000000e+06 -1.010647000000000000e+06 4.383970000000000000e+05 -6.260800000000000000e+04 -6.379490000000000000e+05 6.909230000000000000e+05 1.270807000000000000e+06 -1.828853000000000000e+06 -1.352928000000000000e+06 1.841251000000000000e+06 -9.074670000000000000e+05 -7.849900000000000000e+04
-2.614469000000000000e+06 -2.954812000000000000e+06 1.455844000000000000e+06 -1.295396000000000000e+06 -7.393030000000000000e+05 -1.876736000000000000e+06 2.058844000000000000e+06 6.160880000000000000e+05 -1.010647000000000000e+06 7.998016000000000000e+06 -1.761998000000000000e+06 9.927610000000000000e+05 -1.893181000000000000e+06 4.382620000000000000e+05 -7.764660000000000000e+05 1.181108000000000000e+06 -1.964023000000000000e+06 6.000230000000000000e+05 2.717532000000000000e+06 -3.953730000000000000e+05
2.197126000000000000e+06 9.719040000000000000e+05 -9.718790000000000000e+05 4.829640000000000000e+05 7.133100000000000000e+05 1.443500000000000000e+05 -3.032070000000000000e+05 5.155980000000000000e+05 4.383970000000000000e+05 -1.761998000000000000e+06 6.283924000000000000e+06 -1.427722000000000000e+06 -1.738950000000000000e+05 -2.104677000000000000e+06 6.990690000000000000e+05 -2.492392000000000000e+06 -9.597520000000000000e+05 -8.472060000000000000e+05 -2.859802000000000000e+06 6.449380000000000000e+05
1.965468000000000000e+06 6.823380000000000000e+05 -8.947440000000000000e+05 2.111950000000000000e+05 2.338100000000000000e+04 -8.126180000000000000e+05 2.319858000000000000e+06 -1.392543000000000000e+06 -6.260800000000000000e+04 9.927610000000000000e+05 -1.427722000000000000e+06 7.111405000000000000e+06 1.949852000000000000e+06 6.247710000000000000e+05 9.528910000000000000e+05 9.036350000000000000e+05 -1.322112000000000000e+06 2.384749000000000000e+06 1.360239000000000000e+06 -6.900340000000000000e+05
2.336184000000000000e+06 1.355380000000000000e+06 1.315040000000000000e+05 2.041910000000000000e+06 1.188654000000000000e+06 1.841048000000000000e+06 -1.825408000000000000e+06 -1.632003000000000000e+06 -6.379490000000000000e+05 -1.893181000000000000e+06 -1.738950000000000000e+05 1.949852000000000000e+06 7.920999000000000000e+06 -2.640147000000000000e+06 -1.284866000000000000e+06 -1.745545000000000000e+06 -7.954410000000000000e+05 -3.485775000000000000e+06 -1.540254000000000000e+06 -5.815730000000000000e+05
-2.882240000000000000e+06 -3.490950000000000000e+05 -1.659880000000000000e+06 1.123700000000000000e+06 -2.286279000000000000e+06 -2.065816000000000000e+06 2.962940000000000000e+05 5.809100000000000000e+04 6.909230000000000000e+05 4.382620000000000000e+05 -2.104677000000000000e+06 6.247710000000000000e+05 -2.640147000000000000e+06 7.380057000000000000e+06 2.310400000000000000e+05 -3.158970000000000000e+05 6.601150000000000000e+05 8.339460000000000000e+05 -4.921200000000000000e+05 2.061232000000000000e+06
-8.165580000000000000e+05 3.693500000000000000e+05 2.445942000000000000e+06 1.132003000000000000e+06 -6.995700000000000000e+04 1.776824000000000000e+06 5.189110000000000000e+05 3.054840000000000000e+05 1.270807000000000000e+06 -7.764660000000000000e+05 6.990690000000000000e+05 9.528910000000000000e+05 -1.284866000000000000e+06 2.310400000000000000e+05 5.793247000000000000e+06 -2.791000000000000000e+03 -9.122620000000000000e+05 2.302116000000000000e+06 -1.395723000000000000e+06 -3.513040000000000000e+05
1.426460000000000000e+05 -2.688972000000000000e+06 5.959470000000000000e+05 -1.125240000000000000e+05 -2.171374000000000000e+06 2.228190000000000000e+05 -2.467026000000000000e+06 1.129953000000000000e+06 -1.828853000000000000e+06 1.181108000000000000e+06 -2.492392000000000000e+06 9.036350000000000000e+05 -1.745545000000000000e+06 -3.158970000000000000e+05 -2.791000000000000000e+03 6.271950000000000000e+06 6.455170000000000000e+05 2.183817000000000000e+06 3.098148000000000000e+06 -4.482360000000000000e+05
5.226100000000000000e+05 -2.184907000000000000e+06 1.692314000000000000e+06 -3.881438000000000000e+06 -3.498390000000000000e+05 -1.572740000000000000e+05 -1.435902000000000000e+06 2.437259000000000000e+06 -1.352928000000000000e+06 -1.964023000000000000e+06 -9.597520000000000000e+05 -1.322112000000000000e+06 -7.954410000000000000e+05 6.601150000000000000e+05 -9.122620000000000000e+05 6.455170000000000000e+05 7.112213000000000000e+06 1.357271000000000000e+06 1.223369000000000000e+06 1.862871000000000000e+06
1.222390000000000000e+06 -2.521874000000000000e+06 1.053343000000000000e+06 -7.713710000000000000e+05 2.945100000000000000e+05 -3.136020000000000000e+05 2.269541000000000000e+06 3.097930000000000000e+05 1.841251000000000000e+06 6.000230000000000000e+05 -8.472060000000000000e+05 2.384749000000000000e+06 -3.485775000000000000e+06 8.339460000000000000e+05 2.302116000000000000e+06 2.183817000000000000e+06 1.357271000000000000e+06 7.399761000000000000e+06 1.443786000000000000e+06 1.108636000000000000e+06
8.798570000000000000e+05 -3.889820000000000000e+05 1.667551000000000000e+06 -3.606625000000000000e+06 -2.027497000000000000e+06 -2.257661000000000000e+06 1.247689000000000000e+06 3.499538000000000000e+06 -9.074670000000000000e+05 2.717532000000000000e+06 -2.859802000000000000e+06 1.360239000000000000e+06 -1.540254000000000000e+06 -4.921200000000000000e+05 -1.395723000000000000e+06 3.098148000000000000e+06 1.223369000000000000e+06 1.443786000000000000e+06 9.791695000000000000e+06 2.510999000000000000e+06
5.588720000000000000e+05 -1.569430000000000000e+06 2.019084000000000000e+06 -8.386970000000000000e+05 -1.897100000000000000e+06 -1.345754000000000000e+06 -1.782525000000000000e+06 3.597444000000000000e+06 -7.849900000000000000e+04 -3.953730000000000000e+05 6.449380000000000000e+05 -6.900340000000000000e+05 -5.815730000000000000e+05 2.061232000000000000e+06 -3.513040000000000000e+05 -4.482360000000000000e+05 1.862871000000000000e+06 1.108636000000000000e+06 2.510999000000000000e+06 5.894275000000000000e+06
