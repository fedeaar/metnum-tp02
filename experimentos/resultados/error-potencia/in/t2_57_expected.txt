2.617551521922755241e+07
1.731863635291882232e+07
1.519248540060525946e+07
1.319231821412696503e+07
1.294680196206130274e+07
1.086311716304613277e+07
9.123751406036745757e+06
8.280631073618886992e+06
7.745224016527818516e+06
6.151861011506424285e+06
5.205599894635772333e+06
4.354346294367770664e+06
2.459937443727058824e+06
1.547038354553694604e+06
1.152703179169424344e+06
8.651340614536330104e+05
5.460719079444759991e+05
2.631765122434193036e+05
1.981078650076507765e+05
2.676672210810850743e+02
